osf/models/preprint.py

```diff
@@ -1054,6 +1054,12 @@ def create_waterbutler_log(self, auth, action, payload):
             params=params
         )
 
+    def add_unregistered_contributor(self, *args, **kwargs):
```

Comment:
Overrides the `add_unregistered_contributor` method of `ContributorMixin`. Unlike nodes, this method is actually used by API V2 endpoints for adding unregistered contributors, and seems to be the only current way of adding an unregistered contributor to a preprint.